tests/testthat/test-api-main.R

```diff
@@ -55,11 +55,10 @@ test_that("Multiple targets", {
     "[  LOAD ]",
     "[  READ ]", # sources
     "<  MAKE > processed",
+    "<  MAKE > data.csv",
     "[ BUILD ] data.csv",
     "[  READ ]", # packages
-    "[ BUILD ] processed",
-    "<  MAKE > data.csv",
-    "[    OK ] data.csv")
+    "[ BUILD ] processed")
   expect_equal(length(dat$messages), length(expected))
   expect_equal(substr(dat$messages, 1, nchar(expected)), expected)
 })
```

Comment:
Not everywhere, but at least in some places, e.g.: https://github.com/richfitz/remake/blob/bd664549b2f1e0e211b88f8efa1f12273dc78e7a/tests/testthat/test-remake.R#L194-L212